lib/llm/src/hub.rs

```diff
@@ -20,7 +20,8 @@ const IGNORED: [&str; 3] = [".gitattributes", "LICENSE", "README.md"];
 
 /// Attempt to download a model from Hugging Face
 /// Returns the directory it is in
-pub async fn from_hf(name: &Path) -> anyhow::Result<PathBuf> {
+pub async fn from_hf(name: impl AsRef<Path>) -> anyhow::Result<PathBuf> {
+    let name = name.as_ref();
```

Comment:
Can you ELI5 why this change was made from previously using `&`, to instead implementing `AsRef` + using `.as_ref()`?

Reply:
It broadens the allowable types to include `str`, which implements `AsRef<Path>`.